examples/olives/js/app.js

```diff
@@ -31,3 +31,6 @@ list(document.querySelector('#main'), tasks, stats);
 
 // Same goes for the control UI
 controls(document.querySelector('#footer'), tasks, stats);
+
+// Force auto focus as this seems sometimes flaky
+document.querySelector('[autofocus]').focus();
```

Comment:
This seems a little hacky so we may want to figure things out first. Could it be a framework bug?